Mathlib/Geometry/Manifold/InteriorBoundary.lean

```diff
@@ -21,10 +21,12 @@ Define the interior and boundary of a manifold.
 - `ModelWithCorners.univ_eq_interior_union_boundary`: `M` is the union of its interior and boundary
 - `ModelWithCorners.interior_boundary_disjoint`: interior and boundary of `M` are disjoint
 - `BoundarylessManifold.isInteriorPoint`: if `M` is boundaryless, every point is an interior point
-
+- `ModelWithCorners.Boundaryless.{boundary_eq_empty, of_boundary_eq_empty}`:
```

Comment:
I'd prefer not to use `{` like this. Part of my brain immediately said "that is a really weird name for a universe"!

Reply:
Sure; fixed. (My mental model is shell expansion, akin to `git add `Geometry/Manifold/{Foo,Bar}.lean`.)

Reply:
`doc-gen` linkifier doesn't work that way.